test/export/CodeSystemExporter.test.ts

```diff
@@ -0,0 +1,102 @@
+    const codeSystem = new FshCodeSystem('MyCodeSystem');
+    doc.codeSystems.set(codeSystem.name, codeSystem);
+    const exported = exporter.export();
+    expect(exported.length).toBe(1);
+    expect(exported[0]).toEqual({
+      name: 'MyCodeSystem',
+      id: 'MyCodeSystem',
+      status: 'active',
+      content: 'complete',
+      url: 'http://example.com/CodeSystem/MyCodeSystem'
+    });
+  });
+
+  it('should export a code system with additional metadata', () => {
+    const codeSystem = new FshCodeSystem('MyCodeSystem');
+    codeSystem.title = 'My Fancy Code System';
+    codeSystem.description = 'Lots of important details about my fancy code system';
+    doc.codeSystems.set(codeSystem.name, codeSystem);
+    const exported = exporter.export();
+    expect(exported.length).toBe(1);
+    expect(exported[0]).toEqual({
+      name: 'MyCodeSystem',
+      id: 'MyCodeSystem',
+      status: 'active',
+      content: 'complete',
+      url: 'http://example.com/CodeSystem/MyCodeSystem',
+      title: 'My Fancy Code System',
+      description: 'Lots of important details about my fancy code system'
+    });
+  });
+
+  it('should export each code system once, even if export is called more than once', () => {
+    const codeSystem = new FshCodeSystem('MyCodeSystem');
+    doc.codeSystems.set(codeSystem.name, codeSystem);
+    const exported = exporter.export();
+    expect(exported.length).toBe(1);
+    const exportedAgain = exporter.export();
+    expect(exportedAgain.length).toBe(1);
+  });
+
+  it('should export a code system with a concept with only a code', () => {
+    const codeSystem = new FshCodeSystem('MyCodeSystem');
+    const concept = new FshConcept('myCode');
+    codeSystem.concepts = [concept];
+    doc.codeSystems.set(codeSystem.name, codeSystem);
+    const exported = exporter.export();
+    expect(exported.length).toBe(1);
+    expect(exported[0]).toEqual({
+      name: 'MyCodeSystem',
+      id: 'MyCodeSystem',
+      status: 'active',
+      content: 'complete',
+      url: 'http://example.com/CodeSystem/MyCodeSystem',
+      concept: [{ code: 'myCode' }]
+    });
+  });
+
+  it('should export a code system with a concept with a code, display, and definition', () => {
+    const codeSystem = new FshCodeSystem('MyCodeSystem');
+    const concept = new FshConcept('myCode', 'My code', 'This is the formal definition of my code');
+    codeSystem.concepts = [concept];
+    doc.codeSystems.set(codeSystem.name, codeSystem);
+    const exported = exporter.export();
+    expect(exported.length).toBe(1);
+    expect(exported[0]).toEqual({
+      name: 'MyCodeSystem',
+      id: 'MyCodeSystem',
+      status: 'active',
+      content: 'complete',
+      url: 'http://example.com/CodeSystem/MyCodeSystem',
+      concept: [
+        {
+          code: 'myCode',
+          display: 'My code',
+          definition: 'This is the formal definition of my code'
+        }
+      ]
+    });
+  });
```

Comment:
Can we either make the above two tests each export _two_ concepts or create a new test with multiple concepts?  Usually there will be more than one, and while I'm sure you got it right, it's good to ensure in a test.